wordpress-mobile/WordPress-iOS
# [Stats Refresh] TimeInterval/Period mapping code

This should be the last "big" PR for Stats! 

Sorry for the large size — It's rather verbose code, but most of it is very same-y and boring and not very interesting or clever.

I have half-finished branch of making `StatsPeriodStore` making use of this — but having all this mapping code, switching the `Store` over to the new WPKit API _and_ making it use Core Data is WAY to big to be a single PR, so I'm breaking this up. The rest will come tomorrow :)

To test:
Nothing to really see here yet!
Make sure the tests pass — I'll add more tests specifically targeting this and you'd be able to see it in action with the subsequent PRs :)
